react/features/remote-video-menu/components/AbstractMuteRemoteParticipantDialog.js

```diff
@@ -8,11 +8,12 @@ import {
 } from '../../analytics';
 import { muteRemoteParticipant } from '../../base/participants';
 
+/* eslint-disable no-unused-vars */
```

Comment:
Why?

Reply:
eslint and flow were exclusively not seeing the APP global being used (when one was satisfied, the other was not) - was the only solution I could find in the end

Reply:
How about `declare var APP: Object;`?

Reply:
after further checking not sure how this got here - removed.